### ISSUE TITLE
setup.py doesn't recognize tesseract 4 (the result of tesseract -v is printed in stdout)

### ISSUE THREAD

**Issue by Cryspart** [sirfz/tesserocr#36]
In my tesseract installation (version 4.00), when i do `tesseract -v` the result is printed in the stdout, so the `version` in get_tesseract_version() in setup.py will be always equal to an empty string

A quick fix can be:
```
p = subprocess.Popen(['tesseract', '-v'], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
stdout_version, version = p.communicate()
if version == '':
    version = stdout_version
```

**Comment by sirfz** [comment]
Hello @Cryspart, please feel free to submit a PR.

**Comment by sirfz** [comment]
Thanks for the PR.

Fixed by #37.
